src/cli/actions/hub/push.js

```diff
@@ -79,29 +79,32 @@ async function push () {
   const usernameName = helpers.extractUsernameName(remoteOriginUrl)
 
   try {
-    const postData = {
-      username_name: usernameName,
-      DOTENV_KEYS: dotenvKeysContent,
-      DOTENV_VAULT: dotenvVaultContent
-    }
-    const options = {
+    const response = await request(pushUrl, {
+      method: 'POST',
       headers: {
-        Authorization: `Bearer ${oauthToken}`
-      }
-    }
-    await axios.post(pushUrl, postData, options)
-  } catch (error) {
-    if (error.response && error.response.data) {
-      logger.http(error.response.data)
-      spinner.fail(error.response.data.error.message)
-      if (error.response.status === 404) {
+        Authorization: `Bearer ${oauthToken}`,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        username_name: usernameName,
+        DOTENV_KEYS: dotenvKeysContent,
+        DOTENV_VAULT: dotenvVaultContent
+      })
+    })
+
+    const responseData = await response.body.json()
+
+    if (response.statusCode !== 200) {
```

Comment:
same here. `200` tends to be too strict as a project develops. especially one where an older version of the cli can exist on people's CI/CDs.